tests/results-public-samples

```diff
@@ -169,6 +169,10 @@ passive_top_ja3cli_sha256_distinct = 'dd1d9cc7667f5dc30386c9c21e388737bc9f51297e
 passive_top_ja3cli_sha256_distinct_count = 1
 passive_top_ja3cli_sha256_not_distinct = 'dd1d9cc7667f5dc30386c9c21e388737bc9f51297e9b6537e8261c733e95af46'
 passive_top_ja3cli_sha256_not_distinct_count = 4
+passive_dnsbl_test = 9
+passive_dnsbl_after_update = 18
+list_dnsbl_1 = [('127.0.0.2', 1, '127.0.0.10'), ('127.0.0.2', 1, '127.0.0.2'), ('127.0.0.2', 1, '127.0.0.4'), ('138.36.1.237', 1, '127.0.0.3'), ('138.36.1.237', 1, '127.0.0.4'), ('178.88.114.106', 1, '127.0.0.3'), ('178.88.114.106', 1, '127.0.0.4'), ('181.92.119.72', 1, '127.0.0.11'), ('181.92.119.72', 1, '127.0.0.4')]
```

Comment:
Maybe the names could be changed to  `passive_dnsbl_count_before_update`, `passive_dnsbl_results_before_update`, and same with `after` instead of `before`?